website/docs/d/satellite_location.html.markdown

```diff
@@ -42,7 +42,7 @@ In addition to all argument reference list, you can access the following attribu
     * `zone`- The name of the zone
     * `host_labels`- Host Labels
 - `logging_account_id` - (String) The account ID for IBM Cloud Log Analysis with IBM Cloud Log Analysis log forwarding.
-- `managed_from` - (String) The IBM Cloud metro from which the Satellite location is managed. To list available multizone regions, run `ibmcloud ks locations`. such as `wdc04`, `wdc06`, or `lon04`.
+- `managed_from` - (String) The IBM Cloud regions that you can choose from to manage your Satellite location. To list available multizone regions, run `ibmcloud ks locations`. For more information, refer to [supported IBM Cloud locations](https://cloud.ibm.com/docs/satellite?topic=satellite-sat-regions).
```

Comment:
why this changes are part of this pr

Reply:
proabably, I cloned from the master, and this edit is still in the progress to merge? Not sure, I am guessing Harini. Because this was in earlier PR.